components/property_access.rst

```diff
@@ -394,6 +394,51 @@ and ``removeChild()`` methods to access to the ``children`` property.
 
 If available, *adder* and *remover* methods have priority over a *setter* method.
 
+Using non-standard adder/remover methods
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Like we just said, ``PropertyAccessor`` class allows to update the content of arrays stored in properties through
+*adder* and *remover* methods. But sometimes theses methods doesn't use the standard ``add`` and ``remove`` prefix::
+
+    // ...
+    class PeopleList
+    {
+        /**
+         * @var string[]
+         */
+        private $peoples = [];
+
+        public function getPeoples(): array
+        {
+            return $this->peoples;
+        }
```

Comment:
```suggestion
        // ...
```

I would propose to reduce this code example a bit and only focus on the important things: the custom method names